test/test_core.py

```diff
@@ -5713,6 +5713,8 @@ def test_getdents64_special_cases(self):
     self.do_run_in_out_file_test('fs/test_getdents64_special_cases.cpp')
 
   def test_getcwd_with_non_ascii_name(self):
+    self.emcc_args += ['-sWASMFS']
+    self.emcc_args += ['-sFORCE_FILESYSTEM']
```

Comment:
Instead of forcing this test to use WasmFS, it would be better to use the `@also_with_wasmfs_js` decorator. But looking at the test file, it doesn't look like it even uses the `getcwd` JS API, so we probably don't want to modify this test at all.

Is there another test we can reuse that already uses the JS API? If not, we should add a new test.

Reply:
@tlively  config.yml is calling wasmfs.test_getcwd_with_non_ascii_name so that the reason why I did not write a new test. Is there any other place where this test needs to be called or should I go ahead to write a new test for this.